receiver/gitproviderreceiver/README.md

```diff
@@ -0,0 +1,32 @@
+# Git Provider Receiver
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
+| Distributions | [] |
+| Issues        | ![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fgitprovider%20&label=open&color=orange&logo=opentelemetry) ![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fgitprovider%20&label=closed&color=blue&logo=opentelemetry) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section --> 
+
+The Git Provider receiver scrapes data from Git vendors.
```

Comment:
I'm having some trouble understanding what this means just by looking at the readme. Can you provide a few examples here? Like: 

"When GitHub is used, the following metrics are expected to be created as part of this receiver:"

It's OK to include only a couple of examples, pointing people to the documentation.md file for complete reference.

Reply:
Updated this readme with additional information & regenerated as we've created our own upstream distro of the collector.